### PR TITLE
feature/#CU-a0jmax-Document-Get-bookings-API-Design

### PR DESCRIPTION
**Pull request by lotusmp87**
##Changed:
- Updated Book/ (GET) endpoint to match QT booking management design

### CODE REVIEW

**Comment by lotusmp87** [comment]
Task linked: [CU-a0jmax Document Get bookings API Design](https://app.clickup.com/t/a0jmax)

File: tn_api_pricing_booking_spec.yml [hunk 1]
```diff
@@ -862,14 +857,40 @@ components:
...
+                          items:
+                            $ref: '#/components/schemas/pnr_info'
                         trip_type:
-                            description: Trip types, values include "OneWay", "RoundTrip", "FlexTrip" and "FareStructure"
-                            type: string
-                            example: OneWay 
+                          description: Trip type
+                          type: string
+                          example: multi_city
+                          enum: [one_wWay, round_trip, multi_city]
```
Comment (mcclymont-k) [inline tn_api_pricing_booking_spec.yml]:
```suggestion
                          enum: [one_way, round_trip, multi_city]
```

File: tn_api_pricing_booking_spec.yml [hunk 2]
```diff
@@ -862,14 +857,40 @@ components:
                             type: string
                             description: Unique Identifier of a Universal Record. 
                             example: "AXY67G"
-                        ticketed:       
-                            type: boolean
-                            description: Notes whether the booking has been ticketed. 
-                            example: true
+                        primary_passenger:
```
Comment (Iceman997) [inline tn_api_pricing_booking_spec.yml:860]: Would it make sense to just have 'passenger_first_name' and 'passenger_last_name' at the root level? Would make sorting etc easier

Comment (lotusmp87) [reply tn_api_pricing_booking_spec.yml:860]: The reason I did it like this initially was to make it expandable to passenger details in the next API. But if it's gonna be a pain on QT I can for sure change that.

Comment (mcclymont-k) [reply tn_api_pricing_booking_spec.yml:860]: I think the way it is makes sense if we are looking to add more parameters here in the future

**Review by mcclymont-k — APPROVED** [review]

**Review by Iceman997 — APPROVED** [review]

**Comment by Iceman997** [comment]
In the booking details drawer, there's fields for agent name - would it make sense to return that here in the list level, with agent email etc? 
image

**Comment by lotusmp87** [comment]
> name

We are not currently storing agent name. We can make a card for that addition? @Iceman997

**Pull request merged**